internal/interpreter/dynamic_component.rs

```diff
@@ -1474,7 +1475,7 @@ pub fn instantiate(
         },
     );
 
-    for rep_in_comp in &component_type.repeater {
+    for rep_in_comp in component_type.repeater.values() {
```

Comment:
this used to iterate over the repeated element in order.  now it is in a random order.
This will make the construction of element undetermined.  
Probably good because the user shouldn't rely on the order.
But might make bug reproducing harder because it is non-deterministic.

Reply:
Good point! I'll change that to use a binary tree instead of a hash map.